modules/update-evaluate-availability.adoc

```diff
@@ -19,96 +19,3 @@ If the CVO finds that the cluster does not match the risks of an update, or that
 
 The user interface, either the web console or the OpenShift CLI (`oc`), presents this information in sectioned headings to the administrator.
```

Comment:
do we want to link out from this concept section to "here's the docs for driving those interfaces to consume this infomation"?  Or do only do procedure -> context links, and not context -> proceedure links?

Reply:
It's tricky, do you mean linking to the [CLI](https://docs.openshift.com/container-platform/4.14/updating/updating_a_cluster/updating-cluster-cli.html#update-upgrading-cli_updating-cluster-cli) and [web console](https://docs.openshift.com/container-platform/4.14/updating/updating_a_cluster/updating-cluster-web-console.html#update-upgrading-web_updating-cluster-web-console) update procedures, where there's a step or two showing how to view the available updates?

I can't link inline in this module, it would have to be in an additional resources list at the end of the section. And if you mean to link to these pages, the context might be lost by the time they get to the additional resources section and see links to update procedures. Maybe I can preface the links with "To learn more about viewing available updates, see the following....".